docs/src/nonparametric.md

```diff
@@ -0,0 +1,70 @@
+# Nonparametric tests
+
+## Anderson-Darling test
+
+The null hypothesis of the Anderson–Darling test is that a dataset comes from a certain
```

Comment:
Why describe this test and not the others? Isn't that redundant with the docstrings?

Reply:
I copied this one-to-one from the `.rst` file. Should I delete it?

Reply:
I would say yes: either that's completely redundant with the docstring, or the missing pieces should be moved to the docstring.

Reply:
And same remark for Kolmogorov–Smirnov. To nuance my position, I guess it would be fine to keep a sentence here if it describes what is common to the two AD tests and says a word about their differences (something which isn't always very natural to put in docstrings). But that's not a requirement, and there shouldn't be too many repetitions with the docstrings.

Reply:
Ok, I have replaced them by one-sentence introductions.